templates/house_view/mine_chart.html

```diff
@@ -0,0 +1,28 @@
+{% load static %}
+
+<script type="text/javascript" src="{% static 'js/random_colors.js' %}" ></script>
+<script src="{%static 'js/pie_chart.js'%}" type="text/javascript"></script>
+<script src="{%static 'js/bar_chart.js'%}" type="text/javascript"></script>
+<script>
```

Comment:
Can you please move the script to another file

Reply:
Can I reslove this conversation? @ronyyosef

Reply:
Yes, thanks for explanation!